packages/terra-form-checkbox/tests/jest/CheckboxField.test.jsx

```diff
@@ -8,12 +8,30 @@ import Checkbox from '../../src/Checkbox';
 
 window.matchMedia = () => ({ matches: true });
 
+let userAgentGetter;
+beforeEach(() => {
+  userAgentGetter = jest.spyOn(window.navigator, 'userAgent', 'get');
+});
+
 it('should render a default checkbox field', () => {
   const checkBox = (<CheckboxField legend="Default CheckboxField" />);
   const wrapper = shallowWithIntl(checkBox);
   expect(wrapper).toMatchSnapshot();
 });
 
+it('should render checkbox field with div element for safari browser or Edg browser', () => {
+  userAgentGetter.mockReturnValue('safari Edg');
+  const wrapper = shallowWithIntl(<checkboxField legend="Coustom Message CheckboxField" />);
+  expect(wrapper).toMatchSnapshot();
```

Comment:
We shouldn't rely purely on snapshots for testing as they can be overwritten and incorrect snapshots won't register as failures. There should be an additional expect statement to explicitly test for the div within the wrapper.

Reply:
i have made changes as per suggestions.